tg_bot/modules/reporting.py

```diff
@@ -1,25 +1,24 @@
+# Copyrights HarukaAya :)))))
```

Comment:
lmao, Nice copy paste

Reply:
Doesn't matter tho, their credits are preserved on the file and if it works it's cool.

Tho if original author commits are preserved to this would be even better (and proper way of doing it)

Reply:
will have to change dozen stuff

Reply:
This commit changes all f-strings to .format again

Reply:
and adds some settings

Reply:
Okay drag, do ur coding magic sur :)

Reply:
sure sure